knowage-vue/src/modules/documentExecution/dashboard/widget/WidgetEditor/WidgetEditorSettingsTab/common/WidgetEditorColorPicker.vue

```diff
@@ -22,34 +25,48 @@ export default defineComponent({
     data() {
         return {
             modelValue: null as any,
-            color: '',
-            colorPickTimer: null as any
+            color: '#59c7f9',
+            colorPickerVisible: false,
+            colorPickTimer: null as any,
+            useClickOutside
         }
     },
+    setup() {
+        const knowageStyleIcon = ref(null)
+        let colorPickerVisible = ref(false)
+        let contextMenuVisible = ref(false)
+        useClickOutside(knowageStyleIcon, () => {
+            colorPickerVisible.value = false
+            contextMenuVisible.value = false
+        })
+        return { colorPickerVisible, contextMenuVisible, knowageStyleIcon }
+    },
```

Comment:
Why you had to mix composition and options api? the first ref should be a this.$refs?

Reply:
We tried a lot of things with the color picker, this is the only way we managed to make it work, **useClickOutside** is helping us know where the user clicked so we can close it if needed.
Linter is not complaining about the ref() and seems to work as expected.

Reply:
Unfortunatelly intended.